storybook/qmlTests/TestComponents/tst_DerivatonPathInput.qml

```diff
@@ -0,0 +1,144 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQml 2.14
+import QtTest 1.0
+
+import AppLayouts.Wallet.addaccount.panels 1.0
+import AppLayouts.Wallet.addaccount.panels.DerivationPathInput 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    TestCase {
+        name: "DerivationPathInputControllerTests"
+
+        Component {
+            id: controllerComponent
+
+            Controller {
+                enabledColor: "white"
+                frozenColor: "black"
+                errorColor: "red"
+            }
+        }
+
+        property Controller controller: null
+
+        function init() {
+            controller = createTemporaryObject(controllerComponent, root)
+        }
+
+        readonly property var testDataBases: [
+                {name: "Ethereum Standard", base: "m/44'/60'/0'/0", expected: ["m/44'/", "60", "'/", "0", "'/", "0"]},
+                {name: "Custom", base: "m/44'/", expected: ["m/44'/"]},
+                {name: "Ethereum Ledger", base: "m/44'/60'/0'", expected: ["m/44'/", "60", "'/", "0", "'"]},
+                {name: "Ethereum Ledger Live", base: "m/44'/60'", expected: ["m/44'/", "60", "'"]},
+            ]
+
+        function test_parseRegularBases() {
```

Comment:
For all these TDD test functions, the usual way to pass the data is like this:
(ie the test data comes from a helper function called `test_<func>_data()`)
```qml
    function test_table_data() {
        return [
            {tag: "2 + 2 = 4", a: 2, b: 2, answer: 4 },
            {tag: "2 + 6 = 8", a: 2, b: 6, answer: 8 },
        ]
    }

    function test_table(data) {
        //data comes from test_table_data
        compare(data.a + data.b, data.answer)
    }
```

Reply:
See https://doc.qt.io/qt-6/qml-qttest-testcase.html#data-driven-tests